app/_data/docs_nav_kic_2.5.x.yml

```diff
@@ -33,6 +33,8 @@ items:
         url: /concepts/security
       - text: Ingress Resource API Versions
         url: /concepts/ingress-versions
+      - text: Gateway API
```

Comment:
Do we want to include `/concepts/gateway-api` page in versions 2.4 and before? I think we have already supported some gateway API features in 2.3/2.4.

Reply:
I understand the suggestion, but the alpha maturity builds had no official support so backporting documentation to them would seem to only serve to add support for the alpha builds. Now that we're about to release beta, we _don't want any more feedback on the alpha, we want all new feedback on the beta_ so it would appear to me that we should only focus our efforts on releases that include beta support for now.

Reply:
The Listener behavior discussed in the page is as of 2.5. 2.4 and earlier used a different system that I don't intend to retroactively document.